api/src/main/java/org/openmrs/BaseOpenmrsData.java

```diff
@@ -216,4 +216,4 @@ public void setVoidReason(String voidReason) {
 		this.voidReason = voidReason;
 	}
 	
-}
+}
```

Comment:
Hii @rkunds, a small though considerable change I would like to suggest you is adding a new line at the end of file.

Read more about it [here](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)